src/payment/strategies/cba-mpgs/cba-mpgs-script-loader.ts

```diff
@@ -0,0 +1,22 @@
+import { ScriptLoader } from '@bigcommerce/script-loader';
+
+import { PaymentMethodClientUnavailableError } from '../../errors';
+
+import { CBAMPGSHostWindow, ThreeDSjs } from './cba-mpgs';
+
+export default class CBAMPGSScriptLoader {
+    constructor(
+        private _scriptLoader: ScriptLoader,
+        public _window: CBAMPGSHostWindow = window
```

Comment:
Why is the `window` public here?

Reply:
It makes no sense for it to be public, I'll change it 👍